app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BlogApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+  </head>
+
```

Comment:
- [ ] _User_ and _Post_ **views are missing!** Kindly ensure that your app has corresponding views for each controller action:
```ruby
    # app/views/users/index.html.erb
    # app/views/users/show.html.erb

    # app/views/posts/index.html.erb
    # app/views/posts/show.html.erb
```